PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,24 @@
-## JIRA Link
+### JIRA Link
```

Comment:
minor: can we put a default value under here like below to make it easier to copy paste the link into?:
```
https://project44.atlassian.net/browse/CV-XXXX
```

Reply:
How about `https://project44.atlassian.net/browse/NX-XXXX`? 😉

Reply:
Can we also make it use the markdown link formatting so people don't forget to make it a clickable link. More likely for people to open it if it's clickable.
```
[https://project44.atlassian.net/browse/XXXX](https://project44.atlassian.net/browse/XXXX)
```
Also, I recommend the placeholder not have a dash in it. Its a subtle thing but when you double click a placeholder string with a dash it doesn't select the whole thing. I usually have the JIRA number already copied to my clipboard, so if I could just double click on `XXXX` and paste, I'd be much happier :smile: